Easydict/Swift/Service/Gemini/GeminiService.swift

```diff
@@ -123,11 +122,116 @@ public final class GeminiService: LLMStreamService {
         Defaults[.geminiAPIKey] ?? ""
     }
 
+    override var availableModels: [String] {
+        Defaults[.geminiValidModels]
+    }
+
+    override var model: String {
+        get {
+            Defaults[.geminiModel]
+        }
+        set {
+            // easydict://writeKeyValue?EZGeminiModelKey=gemini-1.5-flash
+            Defaults[.geminiModel] = newValue
+        }
+    }
+
     // MARK: Private
 
     // Set Gemini safety level to BLOCK_NONE
     private let harassmentBlockNone = SafetySetting(harmCategory: .harassment, threshold: .blockNone)
     private let hateSpeechBlockNone = SafetySetting(harmCategory: .hateSpeech, threshold: .blockNone)
     private let sexuallyExplicitBlockNone = SafetySetting(harmCategory: .sexuallyExplicit, threshold: .blockNone)
     private let dangerousContentBlockNone = SafetySetting(harmCategory: .dangerousContent, threshold: .blockNone)
+
+    private func promptContent(
```

Comment:
I suggest moving the prompt code into an extension, such as `GeminiService+Prompt`.